public/client.js

```diff
@@ -17,16 +17,16 @@ const setUsername = () => {
 	socket.emit('set username', $('#userN').val());
 };
 //sends a message
-const sendMessage = (msg) => {
-	// alert(msg);
-	msg = msg.replace(/</g, "&lt;").replace(/>/g, "&gt;");
-	room_name = $(".active").attr("id");
-	socket.emit('Message Request', {
-		msg: msg,
-		room: room_name
-	});
+const sendMessage = (msg)=>  {
+    msg = msg.replace(/</g, "&lt;").replace(/>/g, "&gt;");
+    room_name = $(".active").attr("id");
+    socket.emit('Message Request', {
+        msg: msg,
+        room: room_name
+    });
```

Comment:
If you haven't changed this code, then why it is here :sweat_smile:

Reply:
It's there because it's been formatted. You need to decide on how strict your formatting is going to be and what you're going to use to enforce it. ESLint can do this, as can `prettier`. But, the code itself is the same.